spec/security-warnings-spec.js

```diff
@@ -66,7 +71,7 @@ describe('security warnings', () => {
       }
     })
     w.webContents.once('console-message', (e, level, message) => {
-      assert(message.includes('Node.js Integration with Remote Content'), message)
+      expect(message.includes('Node.js Integration with Remote Content')).to.be.true()
```

Comment:
`expect(message).to.include(''Node.js Integration with Remote Content')` for this and similar!